bigcommerce/products.go

```diff
@@ -60,10 +60,10 @@ type ProductListParams struct {
 // List returns a list of Products matching the given ProductListParams.
 func (s *ProductService) List(ctx context.Context, params *ProductListParams) ([]Product, *http.Response, error) {
 	var products []Product
-	var apiError APIError
+	apiError := new(APIError)
 
 	resp, err := performRequest(ctx, s.sling.New().QueryStruct(params), s.httpClient, &products, apiError) //.Receive(products, apiError)
```

Comment:
`&apiError` would be a simpler solution, but the result would be the same.

Reply:
Thank you for the feedback @klauspost :heart: I'll make that change in a separate PR so that it is made consistently through the project.

Preferably in relation with: https://github.com/corthmann/go-bigcommerce/pull/11